recipes-extended/dpdk/dpdk_19.11-20.04.bb

```diff
@@ -12,10 +14,162 @@ SRC_URI = "git://source.codeaurora.org/external/qoriq/qoriq-components/dpdk;nobr
 "
 SRCREV = "4110a5fed09fa034963cfc246a6285911ecbd540"
 
-include dpdk.inc
+RDEPENDS_${PN} += "python3-core"
+DEPENDS = "virtual/kernel openssl"
+DEPENDS_append_x86-64 = " numactl"
+do_configure[depends] += "virtual/kernel:do_shared_workdir"
+
+inherit module
+
+COMPATIBLE_HOST = '(aarch64|arm|i.86|x86_64).*-linux'
+COMPATIBLE_HOST_libc-musl = 'null'
+COMPATIBLE_HOST_armv4 = 'null'
+COMPATIBLE_HOST_armv5 = 'null'
+COMPATIBLE_HOST_armv6 = 'null'
+
+COMPATIBLE_MACHINE = "(imx|qoriq)"
+
+# override the default value of datadir and let it match dpdk
+# specific installation directory
+export datadir= "${prefix}/share/${BPN}"
+
+export EXAMPLES_BUILD_DIR = "${RTE_TARGET}"
+export PKGS_WITH_OPENSSL_ENABLED = "l2fwd-crypto ipsec-secgw"
+
+DPDK_RTE_TARGET_x86-64 = "x86_64-native-linuxapp-gcc"
+DPDK_RTE_TARGET_x86 = "i686-native-linuxapp-gcc"
+DPDK_RTE_TARGET_armv7a = "${ARCH}-armv7a-linuxapp-gcc"
+DPDK_RTE_TARGET_armv7ve = "${ARCH}-armv7a-linuxapp-gcc"
+DPDK_RTE_TARGET ?= "${ARCH}-dpaa-linuxapp-gcc"
+export RTE_TARGET = "${DPDK_RTE_TARGET}"
+
+export RTE_OUTPUT = "${S}/${RTE_TARGET}"
+export MODULE_DIR = "/lib/modules/${KERNEL_VERSION}/kernel/drivers/net"
+
+S = "${WORKDIR}/git"
```

Comment:
dpdk.inc is a great idea. dpdk-$(VERSION).inc would be bad.
I disagree to this change

Reply:
just merge dpdk.inc to dpdk_${VERSION}.bb, not dpdk-$(VERSION).inc. I get your point. will revert this change.

Reply:
It not merged yet - just take it out (git rebase -i)